searx/plugins/ahmia_filter.py

```diff
@@ -7,7 +7,7 @@
 
 name = "Ahmia blacklist"
 description = "Filter out onion results that appear in Ahmia's blacklist. (See https://ahmia.fi/blacklist)"
-default_on = True
```

Comment:
Despite this value, it should be disabled by default: see the `init` function.
The plugin is actually enabled only when `settings['outgoing']['using_tor_proxy'] is True`

Reply:
Thanks! .. I dropped 906690c and fixed comment about in settings.yml (6debda24 line 163)